### PR TITLE
Add text around example queries to give better context

### PR DESCRIPTION
**Pull request by rleighton**
Add text around example queries to give better context.

### CODE REVIEW

File: docs/gitbook/historical-search/example-queries.md [hunk 1]
```diff
@@ -49,6 +61,9 @@ LIMIT 10
 
 ## Show VPC Flowlog activity for SSH and RDP
 
+Remote shells typically have a human at one end. During an investigation isolating sessions from SSH and RDP is often
+a standard activity to try to identify specific actor activity.
```
Comment (austinbyers) [inline docs/gitbook/historical-search/example-queries.md]: " a standard activity" => "a standard procedure"

File: docs/gitbook/historical-search/example-queries.md [hunk 2]
```diff
@@ -89,6 +117,8 @@ ORDER BY p_event_time ASC
 
 ## Find all console "root" logins in CloudTrail
 
+The root account almost always should not be signing into the AWS console. This will check for instance of that activity.
```
Comment (austinbyers) [inline docs/gitbook/historical-search/example-queries.md]: > The root account should almost never sign into the AWS console; find all such logins

File: docs/gitbook/historical-search/example-queries.md [hunk 3]
```diff
@@ -119,6 +153,10 @@ ORDER BY total_rows DESC
 
 ## Show CloudTrail activity related to an AWS instance
 
+During an investigation a particular instance may become the focus. For example, if it is compromised.
+This query uses the the Panther field p_any_aws_instance_ids to easily search over all CloudTrail events for
```
Comment (austinbyers) [inline docs/gitbook/historical-search/example-queries.md]: Panther field `p_any_aws_instance_ids` (monospace)

File: docs/gitbook/historical-search/example-queries.md (hunk 1 above)
Comment (rleighton) [reply docs/gitbook/historical-search/example-queries.md]: Thanks again for the edits! Fixed.

File: docs/gitbook/historical-search/example-queries.md (hunk 3 above)
Comment (rleighton) [reply docs/gitbook/historical-search/example-queries.md]: Fixed

File: docs/gitbook/historical-search/example-queries.md (hunk 2 above)
Comment (rleighton) [reply docs/gitbook/historical-search/example-queries.md]: Fixed

**Pull request merged**